src/phasorpy/phasor.py

```diff
@@ -0,0 +1,65 @@
+"""
+============================================
+Phasor functions, module: "phasorpy.phasor"
+============================================
+
+This module provides a set of function to 
+compute phasor transform based on the fast 
+fourier transform algorithm from numpy.fft. 
+There are general tools common to FLIM and 
+HSI data, that helps when doing phasor 
+analysis. 
+
```

Comment:
Use [numpy](https://numpydoc.readthedocs.io/en/latest/format.html) and [Python](https://peps.python.org/pep-0257/) docstring conventions, starting with a single line summary. Something like

```
Compute phasor coordinates from fluorescence lifetime and hyperspectral signals.

The ``phasorpy.phasor`` module provides a function to calculate 
phasor coordinates (average intensity, real, and imaginary parts) 
from fluorescence lifetime and hyperspectral imaging using Fourier transform...
```